pymc3/distributions/multivariate.py

```diff
@@ -1940,6 +1940,53 @@ def _distr_parameters_for_repr(self):
         return ["mu"]
 
 
+class CARRV(RandomVariable):
+    name = "car"
+    ndim_supp = 1
+    ndims_params = [1, 2, 1, 1]
+    dtype = "floatX"
+    _print_name = ("CAR", "\\operatorname{CAR}")
+
+    def make_node(self, rng, size, dtype, *dist_params):
```

Comment:
It's better to explicitly specify the parameters (i.e. `mu`, `W`, etc.); otherwise, the signature is unnecessarily ambiguous.  

Also, you'll likely need to convert those distribution parameters to Aesara variables using `aesara.tensor.as_tensor[_variable]`.

Reply:
@brandonwillard, Should I do this convertion in the ```make_node``` method of ```RandomVareable``` instead of the ```dist``` method of CAR ```Distribution``` class?

Reply:
Yeah, it's always good to do that in `Op.make_node`.